libs/execution/src/lib/types/io-types/filesystem.ts

```diff
@@ -13,19 +13,19 @@ import { IOTypeImplementation } from './io-type-implementation';
  */
 export interface FileSystem extends IOTypeImplementation<IOType.FILE_SYSTEM> {
   /**
-   * Retrieves a file from the file system.
+   * Retrieves a file from the file system. Depending on the implementation relative Paths (./) and parent directories (../) are treated differntly.
    * @function getFile
-   * @param {string} filePath - The file path.
-   * @returns {File} - The file or null if the file does not exist.
+   * @param {string} path - The path to the file
+   * @returns {FileSystemFile<unknown> | null} - The file or null if the node does not exist.
    */
-  getFile(filePath: string): BinaryFile | null;
+  getFile(path: string): FileSystemFile<unknown> | null;
 
   /**
-   * Saves a file to the file system.
-   * @function putFile
-   * @param {string} filePath - The file path.
-   * @param {File} file - The file to save.
-   * @returns {FileSystem}
+   * Saves a file to the file system. Depending on the implementation relative Paths (./) and parent directories (../) are treated differntly.
+   * @function putNode
```

Comment:
```suggestion
   * @function putFile
```

Or omit the line entirely, because the function name is obvious from the function signature below.